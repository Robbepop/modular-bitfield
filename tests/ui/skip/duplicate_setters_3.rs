use modular_bitfield::prelude::*;

#[bitfield]
pub struct Sparse {
    #[skip(setters)] #[skip]
    unused_1: B10,
    a: bool,
    #[skip]
    unused_2: B10,
    b: bool,
    #[skip]
    unused_3: B10,
}

fn main() {}
