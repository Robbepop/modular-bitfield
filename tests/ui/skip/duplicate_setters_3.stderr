error: encountered duplicate `#[skip]` attribute for field
 --> tests/ui/skip/duplicate_setters_3.rs:5:24
  |
5 |     #[skip(setters)] #[skip]
  |                        ^^^^

error: duplicate `#[skip]` here
 --> tests/ui/skip/duplicate_setters_3.rs:5:7
  |
5 |     #[skip(setters)] #[skip]
  |       ^^^^
